src/python/ensembl/production/factories/metadata/genomeFactory.py

```diff
@@ -203,12 +230,14 @@ def get_genomes(
         organism_name: graphene.List = [],
         organism_group: graphene.List = [],
         anti_organism_name: graphene.List = [],
+        batch_size: graphene.Int = '50',
+        dataset_status: graphene.String = ['Submitted'],
+        update_dataset_status: graphene.String = None, 
         query_param: graphene.String = None,
 
 ):
     schema = graphene.Schema(query=Query)
-    DATABASE_URI = metadata_db_uri
-    engine = create_engine(DATABASE_URI)
+    engine = create_engine(metadata_db_uri)
```

Comment:
We do have some database connections stuff in ensembl-py. Why not using it?

Reply:
Yes, we could use that, will change accordingly